packages/cli/src/__tests__/ceramic-daemon.test.ts

```diff
@@ -290,6 +290,27 @@ describe('Ceramic interop: core <> http-client', () => {
         expect(StreamUtils.serializeState(docV5Core.state)).toEqual(StreamUtils.serializeState(docV5Client.state))
     })
 
+    it('can get stream contents from /raw_data', async () => {
+      const content1 = {test: 123}
+      const content2 = {test: 456, test2: 'abc'}
+      const content3 = {test2: 'def'}
+      const doc = await TileDocument.create(core, content1)
+
+      await anchorDoc(doc)
+      await doc.update(content2)
+      await anchorDoc(doc)
+      await doc.update(content3)
+      await anchorDoc(doc)
+
+      let returnValue
+      await daemon.rawData(
```

Comment:
It might be a more realistic test to use [fetchJson](https://github.com/ceramicnetwork/js-ceramic/blob/881d7f0f17de820290ba6b5b7f4b19e00d2eed6c/packages/common/src/utils/http-utils.ts#L9) to hit the HTTP endpoint rather than calling into the daemon object directly.

Reply:
Done. I needed to [expose the port](https://github.com/dysbulic/js-ceramic/blob/feat/raw_data-endpoint/packages/cli/src/ceramic-daemon.ts#L142) in the daemon in order to know where to connect. I also [exposed the hostname](https://github.com/dysbulic/js-ceramic/blob/feat/raw_data-endpoint/packages/cli/src/ceramic-daemon.ts#L143) b/c the two generally go together and it looked lopsided w/ just one.